unity/Assets/Scripts/BaseFPSAgentController.cs

```diff
@@ -595,6 +600,13 @@ abstract public class BaseFPSAgentController {
                 this.rotateStepDegrees = action.rotateStepDegrees;
             }
 
+             if (action.snapToGrid && !ValidRotateStepDegreesWithSnapToGrid(action.rotateStepDegrees)) {
+                errorMessage = $"Invalid values 'rotateStepDegrees': ${action.rotateStepDegrees} and 'snapToGrid':${action.snapToGrid}. 'snapToGrid': 'True'a is not supported when 'rotateStepDegrees' is different from grid rotation steps of 0, 90, 180, 270 or 360.";
```

Comment:
looks good to go except slight typo in the error message `'True'a is not`, remove the "a"

Reply:
Done